pupa/scrape/event.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, name, start_time, timezone, location, *,
         self.description = description
         self.status = status
         self.classification = classification
-        self.location = {"name": location, "note": "", "coordinates": None}
+        self.location = location
```

Comment:
`location_name` SGTM; also tests fail on this changeset

On Mon, May 18, 2015 at 2:30 PM, Clayton Dunwell <notifications@github.com>
wrote:

> In pupa/scrape/event.py
> <https://github.com/opencivicdata/pupa/pull/180#discussion_r30534604>:
>
> > @@ -62,7 +62,7 @@ def __init__(self, name, start_time, timezone, location, *,
> >          self.description = description
> >          self.status = status
> >          self.classification = classification
> > -        self.location = {"name": location, "note": "", "coordinates": None}
> > +        self.location = location
>
> The change is because the variable name location is a little confusing
> for initialization. As you can see currently, the variable is setting just
> the name key for the location attribute on the Event model. I agree that
> changing the variable name to location_name would be better for sake of
> clarity and convenience since note + coordinates may often be blank
> anyways. Will submit new PR with method for setting note + coordinates
> attributes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencivicdata/pupa/pull/180/files#r30534604>.
>



-- 
Paul Tagliamonte
Software Developer | Sunlight Foundation